AirCasting/Map/GoogleMapView.swift

```diff
@@ -14,18 +14,20 @@ struct GoogleMapView: UIViewRepresentable {
     @EnvironmentObject var tracker: LocationTracker
     @Binding var placePickerDismissed: Bool
     @Binding var isUserInteracting: Bool
+    var isSessionActive: Bool
```

Comment:
why do we have to spill domain nomenclature here? It would be better to call it "centerLocation" or something - keep the stuff in here as map-related as possible